applications/PFEMapplication/CMakeLists.txt~

```diff
@@ -1,46 +0,0 @@
-set(CMAKE_INCLUDE_CURRENT_DIR ON)
-
-message("**** configuring KratosPFEMApplication ****")
-
-include_directories( ${CMAKE_SOURCE_DIR}/kratos )
-
-## generate variables with the sources
-set( KRATOS_PFEM_APPLICATION_SOURCES     
-	${CMAKE_CURRENT_SOURCE_DIR}/PFEM_application.cpp 
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/kratos_pfem_python_application.cpp 
-  	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_meshers_to_python.cpp 
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_utilities_to_python.cpp 
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_processes_to_python.cpp 
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_python/add_custom_io_to_python.cpp      
-
-	${CMAKE_CURRENT_SOURCE_DIR}/custom_conditions/free_surface_cond2d.cpp
-)
-
-###############################################################
-## define library Kratos which defines the basic python interface
-add_library(KratosPFEMApplication SHARED ${KRATOS_PFEM_APPLICATION_SOURCES})
-target_link_libraries(KratosPFEMApplication KratosCore )
-set_target_properties(KratosPFEMApplication PROPERTIES PREFIX "")
-
-# changing the .dll suffix to .pyd 
-if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-	set_target_properties(KratosPFEMApplication PROPERTIES SUFFIX .pyd)
-endif(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-
-# changing the .dylib suffix to .so 
-if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-	set_target_properties(KratosPFEMApplication PROPERTIES SUFFIX .so)
-endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-
-install(TARGETS KratosPFEMApplication DESTINATION libs )
-
-if(${INSTALL_PYTHON_FILES} MATCHES ON)
-  get_filename_component (CURRENT_DIR_NAME ${CMAKE_CURRENT_SOURCE_DIR} NAME)
-  install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/python_scripts DESTINATION applications/${CURRENT_DIR_NAME}  FILES_MATCHING PATTERN "*.py"  PATTERN ".svn" EXCLUDE)
-endif(${INSTALL_PYTHON_FILES} MATCHES ON)
-
-# get_property(inc_dirs DIRECTORY PROPERTY INCLUDE_DIRECTORIES)
-# message("KratosIncompressibleFluidApplication subdir inc_dirs = ${inc_dirs}")
-
-# Add to the KratosMultiphisics Python module
-install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/PFEMApplication.py" DESTINATION KratosMultiphysics )
```

Comment:
This cannot be deleted...

Reply:
Forget about this, i didn't saw the ~

Reply:
Yes this application was deleted some time ago.

Reply:
@roigcarlo  it stills blocks the pr

Reply:
I am not involved in this review, so i will not say anything about the code.
However as a comment, it would be very good if you avoid this sort of enormous PRs.
A PR should include whenever possible only a few files or specific modifications. In this one you are merging a lot of different PRs into your own branch prior to merging it to the master. This makes the reviewing much more difficult and slower